fastlane/lib/fastlane/actions/install_provisioning_profile.rb

```diff
@@ -0,0 +1,52 @@
+require 'shellwords'
+
+module Fastlane
+  module Actions
+    class InstallProvisioningProfileAction < Action
+      def self.run(params)
+        provisioning_profile_path = params[:provisioning_profile_path]
+        FastlaneCore::ProvisioningProfile.install(provisioning_profile_path)
+      end
+
+      def self.description
+        "Install provisioning profile from inputfile"
+      end
+
+      def self.details
+        "Install provisioning profile from inputfile for current user"
+      end
+
+      def self.authors
+        ["SofteqDG"]
+      end
+
+      def self.category
+        :code_signing
+      end
+
+      def self.is_supported?(platform)
+        [:ios, :mac].include?(platform)
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :provisioning_profile_path,
```

Comment:
Should we maybe check if the path exists here? There is a [check if the copying actually worked](https://github.com/fastlane/fastlane/blob/2545fd0225b85c1204b30061f01a30ebd21d45d3/fastlane_core/lib/fastlane_core/provisioning_profile.rb#L78-L80), but might be nice to catch this even earlier. Some other actions might give ideas how to implement that I guess.

Reply:
Nice catch! I'll add a check for path

Reply:
@janpio 
I've added a check inside of `run` method with additional logic to get absolute path cos i don't know if `FastlaneCore::ProvisioningProfile.install` method can work with relative paths.

Reply:
That additional logic is interesting.

It might be possible to check the path in the options definition already - but I am not 100% sure.

Reply:
Yep, it is possible. So what should i do? I can check the path inside of the options definition and only expand path to absolute inside of `run` method without additional checks.

Reply:
I guess that would be the optimal way to do this for an action.

Reply:
Done